client/src/accountSummary/member/messaging/MessagingContainer.js

```diff
@@ -6,29 +6,58 @@
  * @Description: Member Messaging Conversation Card List Container
  *
  */
-
-import React from 'react';
-import PropTypes from "prop-types";
+import React, {useEffect, useState} from 'react';
 import Confirmation from "../../../common/listings/Confirmation";
 import SelectConversation from "./SelectConversation";
+import MessageService from "../../../services/MessageService";
 
 const MESSAGE = {
     NO_RESULTS: "No results found"
 }
 
-function MessagingContainer(props){
-    const {messageData, messageUser} = props;
+function MessagingContainer(){
+    const [messageData,setMessageData] = useState([]);
+    const [myUserId,setMyUserId] = useState(-1);
+    const [loading, setLoading] = useState(true);
+    const [newLoading, setNewLoading] = useState(true);
+
```

Comment:
Can you add a TODO in here to save the user's `uid` in the redux store

Reply:
Addressed!